dags/general/daily_job.py

```diff
@@ -0,0 +1,73 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
+from dags.airflow_utils import (
+    DATA_IMAGE,
+    clone_repo_cmd,
+    mm_failed_task,
+    pod_defaults,
+    pod_env_vars,
+    xs_warehouse,
+)
+from dags.kube_secrets import (
+    SNOWFLAKE_ACCOUNT,
+    SNOWFLAKE_USER,
+    SNOWFLAKE_PASSWORD,
+    SNOWFLAKE_TRANSFORM_ROLE,
+    SNOWFLAKE_TRANSFORM_SCHEMA,
+    SNOWFLAKE_TRANSFORM_WAREHOUSE,
+    SNOWFLAKE_TRANSFORM_DATABASE,
+)
+
+# Load the env vars into a dict and set Secrets
+env = os.environ.copy()
+env_vars = {**pod_env_vars, **{}}
+
+# Default arguments for the DAG
+default_args = {
+    "catchup": False,
+    "depends_on_past": False,
+    "owner": "airflow",
+    "on_failure_callback": mm_failed_task,
+    "retries": 0,
+    "retry_delay": timedelta(minutes=1),
+    "sla": timedelta(hours=8),
+    "start_date": datetime(2019, 1, 1, 0, 0, 0),
+}
+
+# Create the DAG
+dag = DAG("daily_job", default_args=default_args, schedule_interval="0 12 * * *")
```

Comment:
I think `daily_job` is a little vague. Should probably choose a more applicable name, like `post_nps_data`

Reply:
We can use the same dag for other daily tasks in future, hence a generalised name. 
Task name is "post-nps-feedback".